state-chain/traits/src/liquidity.rs

```diff
@@ -23,6 +23,12 @@ pub trait LpRegistration {
 		address: cf_chains::ForeignChainAddress,
 	);
 
+	/// Get the liquidity refund address for the given account and asset.
+	fn get_liquidity_refund_address(
```

Comment:
So you are saying that if an LP is requesting a deposit address, he should also provide the refund address directly here instead of setting it over the extrinsic in the LP pallet?